installing.md

```diff
@@ -57,7 +57,7 @@ You'll need to modify the file to your environment parameters.
 - The `Group` parameter optionally specifies the group to run the process as. It's not required for operation.
 - The `WorkingDirectory` parameter describes where the service working directory is. This parameter should be set to the root directory where the `augur_view` executables are located.
 - The `ExecStart` parameter describes the program to run when the service is started. For `augur_view`, we want Gunicorn to run on startup. In this instance, we need to break up this parameter into multiple parts:
-    - First, we need the absolute or relative (to the `WorkingDirectory`) path to the Gunicorn executable. In this case, because our virtual environment is in the `augur_view` root directory, we are using the relative path to the executable.
+    - First argument will be absolute path or relative path to Gunicorn executable in our virtual environment. For example, if your virtual environment is in the `augur_view` root directory, the relative path will be `<augur_view directory absolute path>/env/bin/gunicorn`
```

Comment:
For this part of the documentation, we should state that the path must be absolute. The systemd docs make clear that the executable must either be an absolute path, or a simple program name. Since we are assuming Gunicorn will not be installed at the system level, we must direct the user to provide the absolute path to their gunicorn executable in the virtual environment.

Reply:
Yes right, I will correct that. Should I just change the statement to: 
'First argument will be the absolute path to the Gunicorn executable, which is installed within the virtual environment.' ?